Formula/r/rmtrash.rb

```diff
@@ -0,0 +1,22 @@
+class Rmtrash < Formula
+  desc "Move files and directories to the trash"
+  homepage "https://github.com/TBXark/rmtrash"
+  url "https://github.com/TBXark/rmtrash/archive/refs/tags/0.6.5.tar.gz"
+  sha256 "0e26ffb62b5c951728807bb3718faa2ce04bbac775ea0d5a08628f67d6319d48"
+  license "MIT"
+  head "https://github.com/TBXark/rmtrash.git", branch: "master"
+
+  depends_on xcode: ["12.0", :build]
+  depends_on :macos
+  uses_from_macos "swift", since: :big_sur
+
+  def install
+    system "swift", "build", "--disable-sandbox", "-c", "release"
+    bin.install ".build/release/rmtrash"
+  end
+
+  test do
+    assert_match version.to_s, shell_output("#{bin}/rmtrash --version")
+    system bin/"rmtrash", "--help"
```

Comment:
maybe do something like this?

```suggestion
    system bin/"rmtrash", "--force", "non_existent_file"
```

Reply:
Good idea, if I want to add this test case, how should I add a `commit message` to pass the `Commit style` action? Or should I still go with the `force push` method?

Reply:
yeah, you need to squash the commits and do a forced push.